uplink/storage/streams/store.go

```diff
@@ -437,19 +437,24 @@ func (s *streamStore) Delete(ctx context.Context, path Path, pathCipher storj.Ci
 		return err
 	}
 
+	var errlist errs.Group
 	for i := 0; i < int(numberOfSegments(&stream, &streamMeta)-1); i++ {
 		currentPath, err := createSegmentPath(ctx, int64(i), path.Bucket(), encPath)
 		if err != nil {
-			return err
+			errlist.Add(err)
+			continue
 		}
 
 		err = s.segments.Delete(ctx, currentPath)
 		if err != nil {
-			return err
+			errlist.Add(err)
```

Comment:
We are not distinguishing the errors, so the error can be returned in the metainfo or the node but I guess that's worth to carry on.

Reply:
we plan to reorganize deleting process so I believe then we can address this